src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_params.py

```diff
@@ -633,7 +630,10 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
     with self.argument_context('storage file copy start') as c:
         from azure.cli.command_modules.storage._validators import validate_source_uri
 
+        c.register_path_argument(options_list=('--destination-path', '-p'))
```

Comment:
List syntax is preferable to tuple syntax for options list. I wouldn't restart the CI for this, but just FYI moving forward

Reply:
Good to know. Thanks!